share/governance.md

```diff
@@ -148,12 +155,39 @@ role. Appointments last until either the officer steps down, or the unlikely
 and unfortunate situation where the PMC removes the officer. The PMC can add
 more officer roles as it sees fit by modifying this document.
 
-1) Election Officer: The election officer can be on the PMC, as long as they are not running in the election they are officiating
-  * The election officer will be a member of the `election-officer` GitHub group which grants him write access to the plumbing repository.
+1) Election Officer: The election officer can be on the PMC, as long as they are not running in the election they are officiating.
+  * The election officer will be a member of the `election-officer` GitHub group which grants them write access to the plumbing repository.
+
+2) Security Officer: This is the point of contact for external or internal security issues, this person has a published GPG key, and will be the main point of contact for CVE numbers and such.
+
+3) Communications Officer: This is the main point of contact for external and internal publicity and marketing efforts and requests.
+
+### Subcommittees
```

Comment:
The use of subcommittees is to encourage more community involvement which can also increase transparency over five people doing everything. Sharing the work amongst more of the community also helps with burn out.